src/Factory.php

```diff
@@ -104,6 +104,13 @@ public function __construct(LoopInterface $loop = null)
     public function open($filename, $flags = null)
     {
         $filename = $this->resolve($filename);
+
+        if (strpos($filename, '\0') !== false || strpos($filename, "\0") !== false) {
```

Comment:
The `'\0'` seems superfluous, no?

Also minor nit: Should use `\strpos()` consistently.

Reply:
Yeah, it is.